exporter/collector/metricsexporter.go

```diff
@@ -756,15 +804,24 @@ func (m *metricMapper) metricNameToType(name string) string {
 
 func numberDataPointToValue(
 	point pdata.NumberDataPoint,
+	normalizationPoint *pdata.NumberDataPoint,
 ) (*monitoringpb.TypedValue, metricpb.MetricDescriptor_ValueType) {
 	if point.Type() == pdata.MetricValueTypeInt {
+		val := point.IntVal()
+		if normalizationPoint != nil {
+			val -= normalizationPoint.IntVal()
```

Comment:
Our exporter doesn't currently mutate data at all.  We could change to do that, but it would mean the collector copies data before it hits the exporter.